manifests/t/Typora/Typora/0.11.15/Typora.Typora.installer.yaml

```diff
@@ -0,0 +1,53 @@
+PackageIdentifier: Typora.Typora
+PackageVersion: 0.11.15
+InstallerType: inno
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /NORESTART
+  SilentWithProgress: /SILENT /NORESTART
+UpgradeBehavior: install
+FileExtensions:
+  - md
+  - markdown
+  - mmd
+  - mkd
+  - mdwn
+  - mdown
+  - mdx
+  - mdtxt
+  - mdtext
+  - apib
+  - rmarkdown
+  - rmd
+  - txt
+  - text
+Installers:
+  - Architecture: x64
+    Scope: user
+    InstallerUrl: https://typora.io/windows/typora-update-x64-1106.exe
```

Comment:
This url is broken.

Reply:
Thanks for pointing that out. Looks like package author deleted this build due to some critical bug has been introduced. Version 0.11.17 is available now, so I'll update the manifest in another PR to fix this.